packages/core/src/effects/effects.combiner.ts

```diff
@@ -26,5 +27,8 @@ export const combineMiddlewareEffects: MiddlewareCombiner = effects => res => re
   return req$.pipe(...mappedEffects);
 };
 
-export const combineRoutes: RouteCombiner = (path, effects) =>
-  ({ path, effects });
+export const combineRoutes: RouteCombiner = (path, configOrEffects) => ({
+  path,
+  effects: isRouteCombinerConfig(configOrEffects) ? configOrEffects.effects : configOrEffects,
+  middlewares: isRouteCombinerConfig(configOrEffects) ? configOrEffects.middlewares || [] : [],
```

Comment:
I would wrap this into `()` brackets :D I mean it is quite unreadable :D

Reply:
`middlewares: isRouteCombinerConfig(configOrEffects) ? (configOrEffects.middlewares || []) : []